recipes/jemalloc/all/conanfile.py

```diff
@@ -49,9 +53,9 @@ class JemallocConan(ConanFile):
 
     _autotools = None
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    # @property
```

Comment:
I'd leave this so the diff is less clutter + there's no layout so it's kinda required

Reply:
I don't understand. This function/method/property is not used anymore. Why you want me to keep it?